### PR TITLE
#1037 Indigo Toolkit information in the 'About' section that is not relevant for the users

### PR DESCRIPTION
**Pull request by dmalysh**

### CODE REVIEW

File: api/c/indigo/src/indigo.cpp [hunk 1]
```diff
@@ -41,6 +42,15 @@
 
 static _SessionLocalContainer<Indigo> indigo_self;
 
+static const std::string kVersionInfo = R"({
```
Comment (dmalysh) [inline api/c/indigo/src/indigo.cpp:45]: Not sure about the place where we need to store this data

Comment (AlexanderSavelyev) [reply api/c/indigo/src/indigo.cpp:45]: please find where the version is created. The string is populated automatically - but not static

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]

File: api/c/indigo/src/indigo.cpp (hunk 1 above)
Comment (dmalysh) [reply api/c/indigo/src/indigo.cpp]: Updated. Current output result is number of version: 1.14.0

File: api/c/indigo/src/indigo.cpp [hunk 2]
```diff
@@ -51,6 +51,39 @@ CEXPORT const char* indigoVersion()
...
+        std::string version = indigoVersion();
+
+        const std::string digits = "0123456789";
+        const auto parse = [&](const auto& shift) { return version.substr(shift, version.find_first_of("-\\")); };
+        const auto slice = [&](const auto& value) {
+            version.erase(0, value.size() + 1);
+            return value;
+        };
+
+        const std::string major_version = slice(version.substr(0, parse(1).find_last_of(digits)));
```
Comment (AliaksandrDziarkach) [inline api/c/indigo/src/indigo.cpp:67]: This will cause error if minor version consists of more than one digit

**Comment by AliaksandrDziarkach** [comment]
I think this task can be done in more straight and clear way:
1) Fix api/indigo-version.cmake to set variables for majorVersion, minorVersion, devTag, commitHash
2) Fix api/c/indigo/src/indigo_version.h.in to define corresponding constants and to define compilerVersion and compilerPlatform.
3) Add function which combine all this defines into json.

File: api/c/indigo/src/indigo.cpp (hunk 2 above)
Comment (dmalysh) [reply api/c/indigo/src/indigo.cpp]: Fixed.

**Comment by AliaksandrDziarkach** [comment]
> > 3\.
> 
> My previous decision was exactly what you proposed, but I got feedback about I shouldn't change *.cmake files and shouldn't add new constants so I have done it through C++ parse string.

Anyway its was wrong commit - git tag value was not parsed into version vars.

**Review by even1024 — APPROVED** [review]

**Review by even1024 — APPROVED** [review]

**Review by even1024 — APPROVED** [review]

**Pull request merged**